test/benchmarks/reader_results.csv

```diff
@@ -1,6 +1,2 @@
 ,EM,f1,top_n_accuracy,top_n,reader_time,seconds_per_query,passages_per_second,reader,error
```

Comment:
Same here: Shouldn't we keep the actual results for the website here in the PR?

Reply:
Yup, old data restored